lib/validate_encrypt_utils.pm

```diff
@@ -63,13 +124,42 @@ sub verify_number_of_encrypted_devices {
           Dumper($actual_number));
 }
 
+=head2 verify_cryptsetup_message
+
+  verify_cryptsetup_message($expected_message, $actual_message);
+
+=over
+
+=item C<$expected_message>
+  A string with the expected status message of a device
+
+=item C<$actual_message>
+  A string with the actual status message of a device
+
+=back
+
+=cut
 sub verify_cryptsetup_message {
     my ($expected_message, $actual_message) = @_;
     record_info("Assert volumes status", "Verify crypted volume status based on test_data expectations");
     assert_matches(qr/$expected_message/, $actual_message,
         "Message of cryptsetup status does not match regex");
 }
 
+=head2 verify_cryptsetup_properties
+
+  verify_cryptsetup_properties($expected_properties, $actual_properties);
+
+=over
+
+=item C<$expected_properties>
+  A hash reference with the expected LUKS properties
+
+=item  C<$actual_properties>
```

Comment:
Nitpicking warning: ` =item  C<$actual_properties>` has two spaces, let's make it one:  `=item C<$actual_properties>`

If you don't mind :)

Reply:
done here and i checked in other places too. hope i havent missed any